README.md

```diff
@@ -2,21 +2,37 @@
 
 # Deconst documentation
 
-Documentation for the *deconst* project: a continuous delivery pipeline for heterogenous documentation. It's also used to develop deconst itself, like some kind of documentation ouroboros. You can read the documentation itself at [deconst.horse](https://deconst.horse/).
+Documentation for the *deconst* project: a continuous delivery
+pipeline for heterogenous documentation. It's also used to develop
+deconst itself, like some kind of documentation ouroboros. You can
+read the documentation itself at
+[deconst.horse](https://deconst.horse/).
 
 ## Building
 
-To build this documentation standalone, use the [Deconst client](https://github.com/deconst/client). Use the Sphinx preparer and a clone of `https://github.com/deconst/deconst-docs-control` as the control repository.
+To build this documentation standalone, use the [Deconst
+client](https://github.com/deconst/client). Use the Sphinx preparer and a clone
+of `https://github.com/deconst/deconst-docs-control` as the control repository.
 
 ### DNS and TLS
 
-DNS entries for `deconst.horse`, `build.deconst.horse`, `staging.deconst.horse`, and `content.staging.deconst.horse` are managed by Cloud DNS entries in the "drgsites" account. They should be pointed to the appropriate load balancers.
+DNS entries for `deconst.horse`, `build.deconst.horse`, `staging.deconst.horse`,
+and `content.staging.deconst.horse` are managed by Cloud DNS entries in the
+"drgsites" account. They should be pointed to the appropriate load balancers.
```

Comment:
Probably should take a look at pulling some of this out since it lives in the ops playbooks. Do you want me to do that, or would you prefer to look through on a first pass?

Reply:
Preference is to leave this PR just about line length. 

Issue to track: https://github.com/deconst/deconst-docs/issues/262